package.json

```diff
@@ -53,7 +53,7 @@
     "@types/react-test-renderer": "^16.9.1",
     "@types/sanitize-filename": "^1.6.3",
     "antlr4ts-cli": "^0.5.0-alpha.4",
-    "cypress": "^12.8.1",
+    "cypress": "12",
```

Comment:
Any reason for dropping this down to 12?

Reply:
Good catch, happened while upgrading and re-downgrading.